go.mod

```diff
@@ -132,3 +132,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/Microsoft/hcsshim => github.com/ambarve/hcsshim v0.9.1-0.20240118174743-ed3de0e40cae
```

Comment:
Is there a pending PR/change on hcsshim that this PR is dependent on?

Reply:
Yeah that PR is here: https://github.com/microsoft/hcsshim/pull/2002